homeassistant/components/devolo_home_control/switch.py

```diff
@@ -19,13 +19,15 @@ async def async_setup_entry(
     entities = []
     for device in devices:
         for binary_switch in device.binary_switch_property:
-            entities.append(
-                DevoloSwitch(
-                    homecontrol=hass.data[DOMAIN]["homecontrol"],
-                    device_instance=device,
-                    element_uid=binary_switch,
+            # Exclude the binary switch which have also a multi_level_switches here, because they are implemented as light devices now.
```

Comment:
Please break long comments around 88 characters.

Reply:
@MartinHjelmare I made this change in my other PR #36597